upup/models/cloudup/resources/addons/node-termination-handler.aws/k8s-1.11.yaml.template

```diff
@@ -130,6 +130,7 @@ spec:
             - matchExpressions:
               - key: node-role.kubernetes.io/master
                 operator: Exists
+      hostNetwork: true
```

Comment:
This one assumes IMDSv2 hop limit = 1, right? I don't think addons in general does this. Although I personally am in the "hostNetwork is safer than max hop limit > 1" camp, I know this is contentious.

Reply:
Non-host-network pods in an IPv6 cluster can't talk to the IPv4 metadata service.